pages/docs/concepts/message.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Message
+weight: 5
+---
+
+## What is a message?
+A `message` is a method of communication used to transmit or exchange information from a sender to the receiver through the use of `channels`. A single `message` can be consumed by multiple independent receivers, and can be also defined as an _event_, _command_, or _request_. The sender includes a payload containing data that needs to be processed and serialized into a format appropriate for the receiver to process.
```

Comment:
```suggestion
A `message` is a method of communication used to transmit or exchange information from a sender to the receiver through the use of `channels`. A single `message` can be consumed by multiple independent receivers, and can also be defined as an _event_ or _command_. The sender includes a payload containing data that needs to be processed and serialized into a format appropriate for the receiver to process.
```

I suggested to remove `request` in the other comment, maybe you missed it

> The sender includes a payload containing data that needs to be processed and serialized into a format appropriate for the receiver to process

regarding this sentence, I still have doubts

I read `The sender includes a payload containing data`

any immediately, my brain asks questions:
- sender includes payload where?
- payload containing data? but payload is data, 

also `data that needs to be processed and serialized` sounds like it is not the receiver that needs to do it, but it is receiver that is doing it

maybe try to explain if new sentences what you want to explain so we can come up with something. Maybe @alequetzalli can help coming up with some ideas